treadmill-aws/treadmill-aws.spec

```diff
@@ -7,7 +7,7 @@ License:        Apache 2.0
 URL:            https://github.com/Morgan-Stanley/treadmill
 Prefix:         /opt/treadmill
 AutoReqProv:    no
-Requires:       conntrack-tools iproute libcgroup bridge-utils openldap-clients lvm2 lvm2-libs ipset iptables rrdtool bc docker python34 skarnet treadmill-pid1 treadmill-bind treadmill-tktfwd
+Requires:       conntrack-tools iproute libcgroup bridge-utils openldap-clients lvm2 lvm2-libs ipset iptables rrdtool bc docker python34 skarnet treadmill-pid1 treadmill-bind treadmill-tktfwd zookeeper
```

Comment:
i am not sure it should be requirements... it will run without it, just will not be able to start zk. ?